### PR TITLE
Adds more alerts to the confirm_usage script.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/cron/confirm_usage [hunk 1]
```diff
@@ -39,13 +48,39 @@ def main
...
+      alert "#{table.to_s.upcase} KEY-VALUE CREATION: "\
+        "too long (#{delta_sec} seconds [threshold #{threshold_sec}]) since "\
+        "last #{table} creation with column #{k} having value #{v}."
     end
   end
+
+  # Verify emails are being sent. Alert if (a) an email has not been sent for
+  # five minutes and (b) no email has been sent for five minutes.
+  last_sent_sec = Time.now - PEGASUS_DB[:poste_deliveries].where(sent_at: nil).invert.order(:id).limit(1).last[:created_at]
+  last_unsent_sec = Time.now - PEGASUS_DB[:poste_deliveries].where(sent_at: nil).order(:id).limit(1).last[:created_at]
```
Comment (ewjordan) [inline bin/cron/confirm_usage]: So we're ordering by id, then limiting to 1 element of that result, then taking the last element - wouldn't that end up giving you the earliest record, not the latest one? Or am I getting something mixed up?

**Comment by ashercodeorg** [comment]
Ping.

File: bin/cron/confirm_usage (hunk 1 above)
Comment (ewjordan) [reply bin/cron/confirm_usage]: (Oops, sorry for the delay, I forgot to hit "Submit" on this)

Comment (ashercodeorg) [reply bin/cron/confirm_usage]: That would make sense. But seems correct as is...

```
irb(main):001:0> PEGASUS_DB[:poste_deliveries].where(sent_at: nil).order(:id).limit(1).last[:created_at]
=> 2016-11-01 18:10:44 +0000
```

Comment (ewjordan) [reply bin/cron/confirm_usage]: Apart from this query issue, the rest looks good!

Comment (ashercodeorg) [reply bin/cron/confirm_usage]: Oops, I was mixed up. Next commit fixes (also improves), please take another look.

Nice catch!

**Pull request merged**